usr/share/rear/finalize/GNU/Linux/310_migrate_udev_rules.sh

```diff
@@ -6,7 +6,7 @@
 have_udev || return 0
 
 # we treat only these rules
-RULE_FILES=( $( echo /etc/udev/rules.d/*persistent*{names,net,cd}.rules ) )
+RULE_FILES=( $( echo /etc/udev/rules.d/*persistent*{names,net,cd}.rules /etc/udev/rules.d/*eno-fix.rules ) )
```

Comment:
I do not understand why there is that `$( echo ... )` command substitution needed.
I.e. I wonder why it is not just
```
RULE_FILES=( /etc/udev/rules.d/*persistent*{names,net,cd}.rules /etc/udev/rules.d/*eno-fix.rules )
```
as in  usr/share/rear/skel/default/etc/scripts/system-setup.d/55-migrate-network-devices.sh